src/server_manager/ui_components/outline-server-view.html

```diff
@@ -435,7 +426,7 @@ <h3>[[accessKeyRows.length]]</h3>
 
           <div class="access-key-list card-section">
             <!-- header row -->
-            <div class="access-key-row header-row">
+            <div class="access-key-row header-row" hidden$="{{!accessKeyRows.length}}">
```

Comment:
nit: can we use `accessKeyRows.length === 0`? (I suspect not and it's probably not worth a computed property.)

Reply:
That didn't work :/